File: VB/MyWizardControl/MyNextButton/MyButtonPainter.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraWizard
Imports DevExpress.XtraEditors.Drawing
Imports DevExpress.Skins

Namespace WindowsApplication1
	Public Class MyButtonPainter
		Inherits SkinBackButtonPainter
		Public Sub New(ByVal provider As DevExpress.Skins.ISkinProvider)
			MyBase.New(provider)

		End Sub

		Protected Overrides Sub DrawContent(ByVal e As DevExpress.Utils.Drawing.ObjectInfoArgs)
			MyBase.DrawContent(e)
		End Sub
		Protected Overrides Function GetRotationAngle(ByVal e As DevExpress.Utils.Drawing.ObjectInfoArgs) As RotateFlipType
			Return RotateFlipType.Rotate180FlipNone
		End Function
	End Class
End Namespace

File: CS/MyWizardControl/MyNextButton/MyButtonPainter.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraWizard;
using DevExpress.XtraEditors.Drawing;
using DevExpress.Skins;

namespace WindowsApplication1
{
    public class MyButtonPainter : SkinBackButtonPainter
    {
        public MyButtonPainter(DevExpress.Skins.ISkinProvider provider)
            : base(provider)
        {

        }

        protected override void DrawContent(DevExpress.Utils.Drawing.ObjectInfoArgs e)
        {
            base.DrawContent(e);
        }
        protected override RotateFlipType GetRotationAngle(DevExpress.Utils.Drawing.ObjectInfoArgs e)
        {
            return RotateFlipType.Rotate180FlipNone;
        }
    }
}
